### DISCUSSION TITLE
[TCOTD2] Gameplay tweaks branch

### DISCUSSION THREAD

**Discussion by Blue-Shadow** [Realm667/Re-Releases#176] [General]
To let you know, I've just pushed [a branch](https://github.com/Realm667/Re-Releases/tree/gameplay_tweaking) where I'll be working on tweaking weapons and monsters for gameplay purposes.

**Comment by nashmuhandes** [comment]
Can you merge this in? I don't think I'll be touching these parts of the project anymore

**Reply by Tormentor667** [reply]
Yes please :)

**Comment by Blue-Shadow** [comment]
Although I had some uncommitted code that's been sitting there since March, which I just pushed, unfortunately, work on this isn't complete yet. There is still about a third of the monsters left (not including bosses, to which I would only do minor changes, if anything).

With that said, I wouldn't mind hearing your thoughts on the work done so far. Depending on the kind of feedback, it might motivate me to continue, or not...

**Reply by nashmuhandes** [reply]
Ahh, ermmm, here's the situation though: Torm is pushing for a October 31st release, and unfortunately my time now is extremely limited as I am _also_ under publisher pressure to push out an updated build of my game Disdain. The development time I'm able to put into TCOTD2 in the short term is extremely limited -- beyond the critical fixes that Torm requested, I don't think I'll be able to contribute much else, at least for the October 31st release :(

What do you think, Blue-Shadow? If that's too tight of a deadline, then perhaps this shouldn't be merged yet - maybe for an update patch?

(BTW I also spoke with Torm about this -- coop support won't be enabled for the October 31st release too -- too many issues with it that I simply do not have enough time to fix on such short notice)

**Comment by Tormentor667** [comment]
Well, I read through the changes that you've made @Blue-Shadow and for me these sound reasonable and fine. I'd suggest you merge the branch and I am doing another playthrough to make myself an own and certain image :)

**Comment by Blue-Shadow** [comment]
The 31st of October, is it? I thought the deadline was way sooner than this. It should be more than enough time for me to go through the rest of the monsters. Once I'm done, I'll play through the map to see how the changes turn out. If they turn out okay, I'll merge the branch. Cool?

**Comment by nashmuhandes** [comment]
Sounds good! Looking forward to test the rebalances!

**Comment by Tormentor667** [comment]
Perfectly fine, thank you :)

**Comment by Blue-Shadow** [comment]
Alright, I've gone through all of the enemies (except for the final boss) and weapons. I did a play-through on Normal difficulty. Gameplay balance is fine, though I'd recommend adding more health and ammo. It's too tight, especially for a "normal" difficulty.

Anyway, the branch has been merged. Have fun.

**Comment by Tormentor667** [comment]
Awesome, thanks a lot for taking care of all of this. Will do a new playthrough this evening :)
